src/System.Text.Encodings.Web/tests/ConfigurableScalarTextEncoder.cs

```diff
@@ -5,7 +5,7 @@
 using System;
 using System.Text.Encodings.Web;
```

Comment:
This using statement should no longer be necessary. Same for similar cases in other files.

Reply:
@stephentoub No problem, it's my first contribution here and I didn't remember that in c# nesting namespaces can use everything from the layer above. Sorry :sweat_smile: